cmd/kubeadm/app/cmd/init.go

```diff
@@ -434,13 +434,13 @@ func runInit(i *initData, out io.Writer) error {
 		}
 
 		// PHASE 2: Generate kubeconfig files for the admin and the kubelet
-		glog.V(2).Infof("[init] generating kubeconfig files")
+		glog.V(2).Infof("[init] generating KubeConfig files")
```

Comment:
@yagonobre we should do the opposite:
KubeConfig -> kubeconfig.

the official naming is lowercase `kubeconfig` in the docs:

https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/
https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html
https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/
https://kubernetes.io/docs/search/?q=kubeconfig